source/_integrations/tag.markdown

```diff
@@ -113,4 +113,5 @@ When a tag is scanned, the `tag_scanned` event is fired. This event contains the
 | Value | Description |
 | - | - |
 | `tag_id` | Identifier of the tag. Use this to decide what to do.
+| `name` | Name of the tag.
```

Comment:
It should be pointed out that this is not unique to address the concerns from @edenhaus in the implementation PR.